block/block_service.proto

```diff
@@ -62,7 +62,7 @@ package com.hedera.hapi.block;
  * limitations under the License.
  */
 
-option java_package = "com.hedera.hapi.block.stream";
+option java_package = "com.hederahashgraph.api.proto.java";
```

Comment:
I'd prefer to use `com.hedera.hapi.block.stream.protoc` for these, to avoid creating even more uses of a package name we do not want to retain.
More generally, the pattern would be `${package}.protoc`.

Hopefully we can remove these unnecessary and non-matching package directives sooner rather than later.